.github/CODEOWNERS

```diff
@@ -1,5 +1,29 @@
-/.github/ @elastic/doc-leads
-/.github/workflows/ @elastic/docs-engineering
+# Unless a later match takes precedence, someone on the docs team must review all PRs to this repo
+*                           @elastic/docs
 
-/serverless/ @elastic/platform-docs
-/serverless/serverless-changelog.asciidoc @elastic/docs
+# Templates, workflows, and config
+/.github/                   @elastic/doc-leads
+/.github/workflows/         @elastic/docs-engineering
+/acrolinx/                  @elastic/doc-leads
+
+# V3 docs
+/cloud-account/             @elastic/platform-docs
+/deploy-manage/             @elastic/platform-docs
+/explore-analyze/           @elastic/platform-docs
+/extend/                    @elastic/platform-docs
+/get-started/               @elastic/platform-docs
+/manage-data/               @elastic/platform-docs
+/reference/                 @elastic/platform-docs
```

Comment:
we have some shared ownership down below here. there's obs, sec, ingest sections. don't know how picky we want to be about splitting up the folder, but platform shouldn't be the only owner

https://github.com/elastic/docs-content/tree/main/reference

Reply:
Agreed, I think at least the following makes sense. For data-analysis and ingestion-tools they are split across the teams. data-analysis we could subfolder. Ingestion-tools I think maybe should be all obs-docs by the looks of it.

|folder|team|
|---|---|
|reference/data-analysis | ? |
|reference/elasticsearch/clients | platform-docs|
|reference/glossary | docs|
|reference/ingestion-tools | ?|
|reference/observability| obs-docs|
|reference/search |platform-docs|
|reference/security |security-docs|